coalib/testing/LocalBearTestHelper.py

```diff
@@ -52,6 +54,19 @@ class LocalBearTestHelper(unittest.TestCase):
     If you miss some methods, get in contact with us, we'll be happy to help!
     """
 
+    def assertComparableObjectsEqual(self, observed_result, expected_result):
+        for observed, expected in zip_longest(
+                observed_result, expected_result):
```

Comment:
we definitely need to assert for length of the lists. Consider we put `None` either into `observed_result` or `expected_result`, and one of the iterators is too short, then we can't distinguish this case and would actually pass the assertion!
```
assertComparableObjectsEqual([a, None], [a]) # <-- Oops, this passes!
```
(Please make a testcase for this :D)